docs/src/_includes/partials/versions-list.html

```diff
@@ -2,4 +2,5 @@
     {% for item in eslintVersions.items %}
     <li><a href="{{ item.path }}" {% if item.selected %} data-current="true" {% endif %}>{{ item.display }}</a></li>
     {%- endfor -%}
+    <li><a href="https://eslint.org/version-support" data-current="true">Previous Versions</a></li>
```

Comment:
We don't want this to be considered the current version because it just links out.
```suggestion
    <li><a href="https://eslint.org/version-support">Previous Versions</a></li>
```

Reply:
done.